docker-compose.yml

```diff
@@ -6,7 +6,12 @@ services:
     ports:
       - "8078:8078"
     environment:
-      - NEUROAGENT_DB__PREFIX=postgresql+asyncpg://postgres:pwd@postgres:5432/neuroagent
+      - NEUROAGENT_DB__PREFIX=postgresql+asyncpg://
+      - NEUROAGENT_DB__HOST=localhost
+      - NEUROAGENT_DB__PORT=5432
+      - NEUROAGENT_DB__USER=postgres
+      - NEUROAGENT_DB__PASSWORD=pwd
+      - NEUROAGENT_DB__NAME=neuroagent
       - NEUROAGENT_KEYCLOAK__ISSUER=${KEYCLOAK_ISSUER}
       - NEUROAGENT_TOOLS__LITERATURE__URL=https://whatever.com
       - NEUROAGENT_OPENAI__TOKEN=${OPENAI_API_KEY}
```

Comment:
I know this is unrelated...but this env variable is not named like that anymore.


It should be `NEUROAGENT_LLM__OPENAI_TOKEN`